src/Components/Nav/components/Category.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { API } from '../../../config';
+
+function Category(props) {
+  const [categoryList, setCategoryList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/categoryData.json')
+      .then(res => res.json())
+      .then(categoryList => setCategoryList(categoryList.category_lists));
+  }, []);
+
+  // useEffect(() => {
+  //   fetch(`${API}/store/category`)
+  //     .then(res => res.json())
+  //     .then(categoryList => setCategoryList(categoryList.category_lists));
+  // }, []);
+
+  return (
+    <HiddenDiv onMouseLeave={props.handleCategoryOff}>
+      <Category_List className="category-all">
```

Comment:
```suggestion
      <CategoryList className="category-all">
```
카멜케이스로 사용해주시고 클래스 셀렉터 꼭 필요한지 확인해주세요

Reply:
카멜케이스로 수정하였고 클래스 셀렉터 제거하였습니다!